### PR TITLE
Question card

### PR DESCRIPTION
**Pull request by JanaDrieschner**

### CODE REVIEW

**Comment by louisweiser** [comment]
the progress is well done, particularly i like the style from nav bar and profile page

File: answer.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        </section>
+      </section>
+    </main>
+
+    <section class="card__navbar">
+      <footer>
+        <nav>
+          <a href="./profile.html"><img src="user.gif" alt="" /></a>
+          <a href="./bookmark.html"><img src="bookmark.gif" alt="" /></a>
+          <a href="./index.html"><img src="home.svg" alt="" /></a>
```
Comment (lenzloesche) [inline answer.html:40]: you could add the alt attribute values to make your quizz app more accessable.

File: answer.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      <section class="card">
+        <p class="card__text">What kind of a selector selects all elements?</p>
+
+        <section class="card__question"></section>
+
+        <button class="card__question">Hide Answer</button>
+
+        <div>
+          <img class="card__bookmark-two" src="bookmark1.png" alt="" />
```
Comment (lenzloesche) [inline answer.html:22]: instead you could call it `card__bookmark--selected` according to BEM

File: answer.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            <li>#html</li>
+            <li>#flexbox</li>
+            <li>#css</li>
+          </ul>
+        </section>
+      </section>
+    </main>
+
+    <section class="card__navbar">
+      <footer>
```
Comment (lenzloesche) [inline answer.html:36]: i think that the section belongs into the footer and not the other way round.

File: styles.css [hunk 4]
```diff
@@ -41,68 +31,212 @@
   flex-direction:column;
   justify-content: space-around;
   align-items:center;
+  position:relative;
   margin: 30px;
   width: 800px;
```
Comment (lenzloesche) [inline styles.css:36]: a fixed width of 800px is a problem.
we write mobile first, mobile displays are smaller.
you could use % or min-width/max-width for responsiveness.

**Review by lenzloesche — COMMENTED** [review]
nice work.
hope the comments help.

**Pull request merged**